app/frontend/src/views/About.vue

```diff
@@ -1,69 +1,69 @@
 <template>
   <div class="about-layout">
     <v-sheet class="help-highlight pa-5 text-center">
-      <v-row justify="center">
-        <v-col lg="8">
-          <h1 class="my-5">
-            Create and publish forms, and receive submissions from your users
-          </h1>
+      <v-container>
```

Comment:
this is super hard to see what i've changed by looking at this diff.
basically i've added a v-container around each section.